components/TodoItem.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/native'
+import { TouchableOpacity } from 'react-native'
+import Icon from 'react-native-vector-icons/EvilIcons'
+
+const TodoItemContainer = styled.View`
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 10px;
+  padding-bottom: 0;
+`
+
+const TouchableTodoItem = styled.TouchableOpacity`
+  flex-direction: row;
+  align-items: center;
+`
+
+const Circle = styled.View`
+  width: 22px;
+  height: 22px;
+  margin-right: 13px;
+  border-radius: 15px;
+  border-width: 1px;
+  border-color: #fff;
+`
+
+const CheckMark = styled.Text`
+  color: #fff;
+  font-size: 16px;
+  text-align: center;
+`
+
+const TodoText = styled.Text`
+  font-size: 16px;
+  font-family: Courier New;
+  ${prop => prop.completed ? 'color: #bbb; text-decoration: line-through; text-decoration-color: #bbb' : 'color: #fff'};
+`
+
+export const TodoItem = ({ text, item, handleBinPress }) => {
+  const [completed, setCompleted] = useState(false)
+
+  //Function used to toggle if a todo item is completed or not, used to render different styling
+  const toggleCompleted = () => {
+    { completed ? setCompleted(false) : setCompleted(true) }
+  }
```

Comment:
you could also do:
`setCompleted(!completed)`
that will "invert" the value, so if it's false, will be set to true and vice versa.

Reply:
Ah, nice!